lisa/sut_orchestrator/azure/tools.py

```diff
@@ -115,7 +115,11 @@ def _get_configuration(self, force_run: bool = False) -> Dict[str, str]:
         waagent_conf_file = self._get_waagent_conf_path()
 
         config = {}
-        cfg = self.node.tools[Cat].run(waagent_conf_file, force_run=force_run).stdout
+        cfg = (
+            self.node.tools[Cat]
+            .run(waagent_conf_file, force_run=force_run, sudo=True)
+            .stdout
+        )
```

Comment:
thanks for the fix, could you please use `cat.read`

Reply:
done